src/main/java/edu/aau/groupc/canteenbackend/auth/Auth.java

```diff
@@ -8,7 +8,7 @@
 @Data
 @Entity
 @Table(name = "auth", uniqueConstraints = {
-        @UniqueConstraint(columnNames = {"username", "token"}),
+        @UniqueConstraint(columnNames = {"token"}),
```

Comment:
Not sure about this one. The idea was to make the **combination of username and token ** unique. Maybe even my current solution was not correct.